### DISCUSSION TITLE
bin en modulo Field

### DISCUSSION THREAD

**Discussion by FranciscoGarciaHorna** [mercadopago/sdk-js#74] [General]
Como obtengo el bin de una tarjeta si estoy usando el modulo field(cardNumber), para posteriormente obtener las todas las cuotas disponibles.
Gracias.

**Comment by lucmantovani** [comment]
Hola, @FranciscoGarciaHorna!

Si estás usando la integración avanzada, puedes obtener el BIN mediante el uso del callback [`binChange`](https://github.com/mercadopago/sdk-js/blob/main/API/fields.md#field-instanceonevent-callback)
Si estás usando la integración por Card Form, no deberías necesitar acceso al BIN ya que el próprio Card Form hace el flujo de busca de cuotas
